### DISCUSSION TITLE
multiple errors out of the box. clangd lsp is not pointing out any error in my c code; tree sitter not working properly, lsp not working and backline

### DISCUSSION THREAD

**Discussion by ash77ash77** [LazyVim/LazyVim#4668] [Q&A] [answered]
![Screenshot 2024-10-30 1 50 13 PM](https://github.com/user-attachments/assets/eb97699e-58ba-4596-b0de-3a2c1840f180)
![Screenshot 2024-10-30 1 49 35 PM](https://github.com/user-attachments/assets/e0c9aefd-ea85-4733-a348-7be5e54d5fa1)
out of the box i'm faced handful of errors. no matter how much I try I cant seem to get clangd lsp working. I can get snippets working with no discription. but JDLTS works just fine with zero issues. clangd LSP doesn't even seem to start. on top of that i'm getting errors like the following
Failed to run config for indent-blankline.nvim

.../share/nvim/lazy/indent-blankline.nvim/lua/ibl/utils.lua:22: tab_char: expected table: 0x7f8ffa953650, got string (│)

# stacktrace:
  - vim/shared.lua:0 _in_ **validate**
  - /indent-blankline.nvim/lua/ibl/utils.lua:22 _in_ **validate**
  - /indent-blankline.nvim/lua/ibl/utils.lua:34 _in_ **validate_config**
  - /indent-blankline.nvim/lua/ibl/config.lua:131 _in_ **validate_config**
  - /indent-blankline.nvim/lua/ibl/config.lua:275 _in_ **set_config**
  - /indent-blankline.nvim/lua/ibl/init.lua:57 _in_ **setup*
  this is one of the many errors i'm facing. some errors happen out of the box some happen after I enable some LSPs like clangd

**Comment by dpetka2001** [comment, marked as answer]
Use Neovim stable.

**Reply by ash77ash77** [reply]
thanks a lot mate. it took me half a day to figure out how to fix this. now i can finally sleep after 2 weeks of agonizing pain 👍 👍 👍

**Reply by dpetka2001** [reply]
You were probably on an old Neovim nightly version and there were changes in the plugin that caused the error. If you also updated to latest Neovim nightly I believe the error should go away (except if the plugin did not update yet to meet the latest Neovim nightly breaking changes). 

Either way if you use Neovim nightly you're likely to face some breaking changes from time to time and you just have to wait until they get resolved or use Neovim stable instead.
